### ISSUE TITLE
Upcoming client status change message not displayed in the client Notes tab

### ISSUE THREAD

**Issue by lamontfr** [savoirfairelinux/sous-chef#812]
### Expected Behaviour
- When I view the client information, I should see the upcoming status change message at the top right in all tabs (personal, billing, status, preferences, notes, orders).

### Actual Behaviour
- When I view the client information, I see the upcoming status change message at the top right in tabs : personal, billing, status, preferences, orders, but I do not see it in the notes tab.

### Steps to reproduce
- Clients / click on a client / click on pull down next to yellow status at top right / set a change of status for a future date / click OK/ notice upcoming change message at top right / click Notes tab / notice that there is no upcoming change message at top right / click Preferences tab / notice upcoming change message at top right

**Comment by JeanFrancoisFournier** [comment]
right! As a plus, it would be nice to see all scheduled status changes for a client (i.e. Paused on September 3rd / Active on September 21st)
